tests/event-log/event-log-level.spec.ts

```diff
@@ -25,15 +25,15 @@ describe('EventLogLevel Tests', () => {
   });
 
   it('separates events by tenant', async () => {
-    const { requester, message } = await TestDataGenerator.generateRecordsWrite();
+    const { author, message } = await TestDataGenerator.generateRecordsWrite();
     const messageCid = await Message.getCid(message);
-    const watermark = await eventLog.append(requester.did, messageCid);
+    const watermark = await eventLog.append(author.did, messageCid);
 
-    const { requester: requester2, message: message2 } = await TestDataGenerator.generateRecordsWrite();
+    const { author: requester2, message: message2 } = await TestDataGenerator.generateRecordsWrite();
```

Comment:
```suggestion
    const { author: author2, message: message2 } = await TestDataGenerator.generateRecordsWrite();
```

Reply:
Okay, I know why missed all the above hits... I had "match whole world" turned on in my search, which ignores any hits with prefixes and suffixes... changing now!